src/main/java/frc/robot/subsystems/vision/VisionIOReal.java

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.vision;
+
+import edu.wpi.first.math.Matrix;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.math.numbers.N1;
+import edu.wpi.first.math.numbers.N3;
+import edu.wpi.first.math.numbers.N8;
+import edu.wpi.first.wpilibj.RobotController;
+import frc.robot.subsystems.vision.Vision.VisionConstants;
+import java.util.Optional;
+import org.photonvision.EstimatedRobotPose;
+import org.photonvision.PhotonCamera;
+
+/** Add your docs here. */
+public class VisionIOReal implements VisionIO {
+  // constants
+  public String cameraName;
+  public PhotonCamera camera;
+  public Matrix<N3, N3> cameraMatrix;
+  public Matrix<N8, N1> distCoeffs;
+  private final VisionConstants constants;
```

Comment:
would thse constants differ from robot to robot

Reply:
they differ from camera to camera because they are lens calibration, ids, etc.